Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 gemspec
 
+gem "rack", "~> 2" # Rack 3 is not compatible with Jasmine and fails -- jasmine-3.99.0/lib/jasmine/server.rb:12:in `start': uninitialized constant Rack::Server (NameError)
```

Comment:
Oh, thank you

Reply:
@ashkulz @mathieujobin  My team is using Rack 3. If your lock it at Rack 2 it would break our system